src/flotilla/agent/db.py

```diff
@@ -23,13 +24,14 @@ class FlotillaAgentDynamo(object):
     """
 
     def __init__(self, instance_id, service_name, status_table,
-                 assignments_table, revisions_table, units_table):
+                 assignments_table, revisions_table, units_table, kms):
```

Comment:
Doh. yeah that's going to break stuff.

I'll take #1 and integration test this over xmas break (catching this --^ and probably some IAM fixes too: no permissions granted against the key atm).